src/exports/global.js

```diff
@@ -1,34 +1,34 @@
 define( [
 	"../core"
-], function( jQuery, noGlobal ) {
+], function( jQuery, noGlobal, globalObject ) {
 
-"use strict";
+        "use strict";
```

Comment:
Please don't change the indentation here.

Also, we use tabs for indentation, not spaces, please change that in the PR as I see you're using spaces everywhere.